components/cam/src/physics/crm/samxx/boundaries.cpp

```diff
@@ -0,0 +1,5 @@
+#include "boundaries.h"
+
+void boundaries(int flag) {
+  periodic(flag);
+}
```

Comment:
Why not just put the contents of "periodic" right here in boundaries?

Reply:
Similarity with the original Fortran code organization.